### PR TITLE
Update StateOutput

### PR DESCRIPTION
**Pull request by JFKFred**

### CODE REVIEW

File: client/src/server/validations.ts [hunk 1]
```diff
@@ -76,5 +76,5 @@ export async function validateMetadata(
 
   console.timeEnd(`timeToValidate`);
 
-  return result;
+  return JSON.parse(JSON.stringify(result)) as Promise<ValidatorResults>;
```
Comment (pChausseC) [inline client/src/server/validations.ts:79]: why stringify and json parse and type cast? is `result` not a an object of type ValidatorResults already?

Comment (pChausseC) [reply client/src/server/validations.ts:79]:
```suggestion
  return result;
```

**Review by pChausseC — CHANGES_REQUESTED** [review]

File: client/src/server/validations.ts (hunk 1 above)
Comment (JFKFred) [reply client/src/server/validations.ts:79]: becuase the reponse (ZodError) add shit:
maybe i can do this backend  ?

[
  {
    validatorId: 'key-anvil-casing',
    validationError: ZodError: [
      {
        "code": "custom",
        "message": "Some attribute's keys do not adhere to Title Case formatting",
        "path": [
          "attributes",
          "background"
        ]
      },
      {
        "code": "custom",
        "message": "Some attribute's keys do not adhere to Title Case formatting",
        "path": [
          "attributes",
          "mouth"
        ]
      },
      {
        "code": "custom",
        "message": "Some attribute's keys do not adhere to Title Case formatting",
        "path": [
          "attributes",
          "nose"
        ]
      },
      {
        "code": "custom",
        "message": "Some attribute's keys do not adhere to Title Case formatting",
        "path": [
          "attributes",
          "Body color"
        ]
      }
    ]
        at KeyAnvilCase.Logic (webpack-internal:///(action-browser)/./node_modules/@ada-anvil/metadraft-validator/esm/rules/key-anvil-case.js:60:26)
        at KeyAnvilCase.Execute (webpack-internal:///(action-browser)/./node_modules/@ada-anvil/metadraft-validator/esm/rules/key-anvil-case.js:46:21)
        at Validator.Execute (webpack-internal:///(action-browser)/./node_modules/@ada-anvil/metadraft-validator/esm/core.js:139:43)
        at validateMetadata (webpack-internal:///(action-browser)/./src/server/validations.ts:56:23)
        at endpoint (webpack-internal:///(action-browser)/./node_modules/next/dist/build/webpack/loaders/next-flight-action-entry-loader.js?actions=%5B%5B%22C%3A%5C%5CUsers%5C%5Cfredg%5C%5COneDrive%5C%5CDocuments%5C%5CCode%5C%5CANVIL%5C%5CMetaDraft%5C%5Cclient%5C%5Csrc%5C%5Cserver%5C%5Cvalidations.ts%22%2C%5B%22validateMetadata%22%5D%5D%5D&__client_imported__=true!:8:17)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:418
        at async r_ (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:38:8148)
        at async r8 (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:41:1256)
        at async doRender (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:1420:30)
        at async cacheEntry.responseCache.get.routeKind (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:1581:28)
        at async DevServer.renderToResponseWithComponentsImpl (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:1489:28)
        at async DevServer.renderPageComponent (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:1920:24)
        at async DevServer.renderToResponseImpl (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:1958:32)
        at async DevServer.pipeImpl (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:917:25)
        at async NextNodeServer.handleCatchallRenderRequest (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\next-server.js:272:17)
        at async DevServer.handleRequestImpl (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\base-server.js:813:17)
        at async C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\dev\next-dev-server.js:339:20
        at async Span.traceAsyncFn (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\trace\trace.js:154:20)
        at async DevServer.handleRequest (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\dev\next-dev-server.js:336:24)
        at async invokeRender (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\lib\router-server.js:173:21)
        at async handleRequest (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\lib\router-server.js:350:24)
        at async requestHandlerImpl (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\lib\router-server.js:374:13)
        at async Server.requestListener (C:\Users\fredg\OneDrive\Documents\Code\ANVIL\MetaDraft\client\node_modules\next\dist\server\lib\start-server.js:141:13) {
      issues: [Array],
      addIssue: [Function (anonymous)],
      addIssues: [Function (anonymous)],
      errors: [Array]
    }
  }
]

**Comment by JFKFred** [comment]
type StateOutput changes...

**Pull request closed without merging**